core/node/eigenda_proxy/proto/common/common.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+package common;
+option go_package = "github.com/Layr-Labs/eigenda/api/grpc/common";
```

Comment:
Do we need this go package?
Same with other files

Reply:
I added the proto files just to know where the compiled code comes from.